### DISCUSSION TITLE
How to return methods as a function?

### DISCUSSION THREAD

**Discussion by dvgmdvgm** [vfsfitvnm/frida-il2cpp-bridge#127] [Q&A]
Yo!:)
Have some methods that cooperate with other one..
class1 "DataController"
method1 "RageLevel"
class2 "StartPosition"
method2 "PlayerIndex"

For example some FridaJS code:     
Just declare "RageLevel" as function:
```
classes.DataController.get_RageLevel = new NativeFunction((0x01a0f090),['void'],['pointer']);
	classes.DataController.RageLevel= function(obj_ptr){
		return this.get_RageLevel(obj_ptr);
};
```
Same with PlayerIndex:
```
classes.StartPosition.get_PlayerIndex = new NativeFunction((0x01e529d0),['void'],['pointer', 'int']);
	classes.StartPosition.PlayerIndex= function(obj_ptr, index){
		return this.get_PlayerIndex(obj_ptr, index);
};
```
And now i can get RageLevel value for specific player by index like this:
```
var _playerIndex = Memory.readPointer(classes.DataController.RageLevel(
        	                                    classes.StartPosition.PlayerIndex(1)).add('0xc0'))
```

It is possible to make something like that with `frida-il2cpp-bridge`?

**Comment by vfsfitvnm** [comment]
It's not clear what your task is, but if you are asking if it's possible to invoke a function within another one, the answer is of course you can

**Comment by dvgmdvgm** [comment]
@vfsfitvnm even if it is not static methods ?
I want to get RageLevel value but only for specific player by index. And want to call this function anytime i want.
Upper just wrote some example how i can achieve it by default frida script. But idk how to make something simillar with `frida-il2cpp-bridge`

Any example please?

**Comment by vfsfitvnm** [comment]
I apologize but I won't write fresh code for you. Please figure out what you have to do, look at the other issues, there are plenty of code snippets...
